src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java

```diff
@@ -1,16 +1,5 @@
 package com.github.dockerjava.netty;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import java.io.IOException;
-import java.net.InetAddress;
-import java.net.InetSocketAddress;
-import java.net.SocketAddress;
```

Comment:
Sorry, bad rebase, will redo once more from scratch/cherry-pick, lets close this one.